### ISSUE TITLE
HammerCommandsTestCase::test_positive_all_options

### ISSUE THREAD

**Issue by ntkathole** [SatelliteQE/robottelo#6018]
```
E           AssertionError: 
E           hammer arf-report delete
E             Added options:
E               * location-id
E               * organization
E               * location
E               * organization-id
E           
E           hammer arf-report download
E             Added options:
E               * location-id
E               * organization
E               * location
E               * organization-id
E           
E           hammer arf-report download-html
E             Added options:
E               * location-id
E               * organization
E               * location
E               * organization-id
E           
E           hammer arf-report info
E             Added options:
E               * location-id
E               * organization
E               * location
E               * organization-id
E           
E           hammer arf-report list
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer audit (new command)
E             Added subcommands:
E               * info
E               * list
E             Added options:
E               * help
E           
E           hammer audit info (new command)
E             Added options:
E               * location-id
E               * location
E               * id
E               * organization
E               * location-title
E               * organization-title
E               * help
E               * organization-id
E           
E           hammer audit list (new command)
E             Added options:
E               * location-id
E               * search
E               * location
E               * page
E               * order
E               * organization
E               * location-title
E               * organization-title
E               * per-page
E               * help
E               * organization-id
E           
E           hammer auth-source
E             Added subcommands:
E               * list
E           
E           hammer auth-source ldap create
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer auth-source ldap delete
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer auth-source ldap info
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer auth-source ldap update
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer auth-source list (new command)
E             Added options:
E               * location-id
E               * search
E               * location
E               * page
E               * order
E               * organization
E               * location-title
E               * organization-title
E               * per-page
E               * help
E               * organization-id
E           
E           hammer bootdisk generic
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer bootdisk host
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer bootdisk subnet
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer capsule create
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer capsule delete
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer capsule import-classes
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer capsule info
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer capsule refresh-features
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer capsule update
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource create
E             Added options:
E               * location-id
E               * use-v4
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource delete
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource image available
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E             Removed options:
E               * name
E               * id
E           
E           hammer compute-resource image create
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource image delete
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource image info
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource image list
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource image update
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource info
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource networks
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer compute-resource update
E             Added options:
E               * location-id
E               * use-v4
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer config-group create
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer config-group delete
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer config-group info
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer config-group list
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer config-group update
E             Added options:
E               * location-id
E               * location
E               * organization
E               * location-title
E               * organization-title
E               * organization-id
E           
E           hammer content-view create
E             Added options:
E               * auto-publish
```

**Comment by ntkathole** [comment]
Now error changes to

```
>                   contents['options'][-1]['help'] += (
                        u' {0}'.format(match.group('help'))
                    )
E                   IndexError: list index out of range

robottelo/cli/hammer.py:130: IndexError
```

**Comment by sghai** [comment]
@surajbora1 : hey would you be able to take this up and fix ? You can reach out to @ntkathole for more info.

**Comment by surajbora1** [comment]
@sghai Started working on it.

**Comment by surajbora1** [comment]
Fixed #6519
